internal/sendconfig/common_workflows.go

```diff
@@ -69,12 +71,14 @@ func UpdateKongAdminSimple(ctx context.Context,
 	// apply the configuration update in Kong
 	timedCtx, cancel := context.WithTimeout(ctx, proxyRequestTimeout)
 	defer cancel()
+	start := time.Now()
 	configSHA, err := PerformUpdate(timedCtx,
 		deprecatedLogger, &kongConfig,
 		kongConfig.InMemory, enableReverseSync,
 		targetConfig, kongConfig.FilterTags, nil, lastConfigSHA, false,
 	)
 	if err != nil {
+		promMetrics.ConfigPassCounter.Inc()
```

Comment:
We want to increment the failure counter here, correct? This block runs when the error from PerformUpdate is non-nil.
```suggestion
		promMetrics.ConfigFailureCounter.Inc()
```